Lib9c/Action/AccountStateDeltaExtensions.cs

```diff
@@ -199,13 +199,8 @@ public static AvatarState GetAvatarState(this IAccountStateDelta states, Address
             }
             if (!agentState.avatarAddresses.ContainsValue(avatarAddress))
             {
-                Log.Error(
-                    "The avatar {0} does not belong to the agent {1}.",
-                    avatarAddress.ToHex(),
-                    agentAddress.ToHex()
-                );
-
-                return false;
+                throw new AgentStateNotContainsAvatarAddressException(
```

Comment:
i think this change is incorrect. (throw exception in `TryGetAvatarStates`)

Reply:
I guess.. 
this line seems to have thrown an exception when there is no AvatarAddress in AgentState.

Reply:
Oh right. It would be correct not to throw an exception inside `TryXxx()`.